doc/source/myrocks/zenfs.rst

```diff
@@ -13,99 +13,191 @@ a Linux kernel implementation which supports NVMe Zoned Namespaces. The kernel
 
-To install the packages, run the following command:
-   
-.. code-block:: bash
-    
-    $ sudo apt install percona-server-server-zenfs
-    $ sudo apt install percona-server-rocksdb-zenfs
-
-Running the Installation
-----------------------------
-
-The following steps successfully run Percona Server with MySQL on the  ``--rocksdb-fs-uri=zenfs://dev:<short_block_device_name>``:
+The following procedure installs and configures Percona Server for MySQL on the  ``--rocksdb-fs-uri=zenfs://dev:<short_block_device_name>``.
 
 .. note::
 
-    The ``<block_device_name>`` can have a short name desigination which is the ``<short_block_device_name>``. For example, the ``block_device_name`` is ``/dev/nvme1n2`` and the short name is ``nvme1n2``.
-
-#. Identify your ZBD device, ``<block_device_name>``, with ``lsblk``. Use the ``-o`` option to specify which columns are returned. The ``NAME``, ``SIZE``, and ``ZONED`` parameters display the block device names, size, and if the device uses the zone model. A ZBD device is identified as ``host-managed``.
-
-    .. sourcecode:: bash
-
-     lsblk -o NAME,SIZE,ZONED
-     NAME        SIZE  ZONED
-     sda       247.9G  none
-     |-sda1    230.9G  none
-     |-sda2        1G  none
-     |-sda3       16G  none
-     sdb        15.5T  host-managed
-
-
-#. Create an auxiliary directory for the ZenFS, for example, ``/var/lib/mysql_aux``. The ZenFS auxiliary directory is a regular (POSIX) file system used internally to resolve file locks and shared access. There are no strict requirements for the auxiliary directory location but the directory must be write accessible for the `mysql:mysql` UNIX system user account. Each ZBD must have an individual auxiliary directory.
-
-
-#. Run ``/zenfs mkfs --zbd <short_block_device_name> --aux_path=<path_to_aux_zenfs_dir>`` in the context of the mysql:mysql account to initialize ZenFS on the ZBD.
-
-    .. sourcecode:: bash
-
-        zenfs mkfs --zbd=nvme1n2 --aux_path=/var/lib/mysql_aux
-
-    .. note::
-
-        This step initializes the metadata on the block device, ``<block_device_name>``, and creates the ``<path_to_aux_zenfs_dir>`` owned by the `mysql:mysql` account. For example, the ``<block_device_name>`` could be ``/dev/nvme1n2``. The ``<path_to_aux_zenfs_dir>`` is a unique, dedicated directory which allows read and write access to `mysql:mysql`.
-
-        If you are re-initializing an already initialized ZBD, you must manually clear the ``/var/lib/mysql_aux`` directory and add the additional ``--force`` parameter.
-
-        .. sourcecode:: bash
-
-            zenfs mkfs --zbd=nvme1n2 --aux_path=/var/lib/mysql_aux  --force
-
-
+   The ``<block_device_name>`` can have a short name designation which is the ``<short_block_device_name>``. For the purposes of this document the ``block_device_name`` is ``/dev/nvme0n2`` and the short name is ``nvme0n2``. 
+    
+The primary supported platform is `Debian 11.1 <https://www.debian.org/releases/bullseye/debian-installer/index>`__.
```

Comment:
Let's be precise what we mean by "supported" here - the only platform that we (Percona) provide a binary release for. There are other Linux OSes that now provide full support for ZenFS, but we don't necessarily provide installable packages for these (yet). We should add a pointer to https://zonedstorage.io/docs/distributions/linux so users can educate themselves about the state of support for various Linux distributions.

Reply:
Done.